tests/io/test_openff.py

```diff
@@ -191,7 +190,7 @@ def test_assign_partial_charges(mol_files):
     openff_mol, atom_map = add_conformer(openff_mol, geometry)
     partial_charges = np.load(mol_files["CCO_charges"])
     openff_mol = assign_partial_charges(openff_mol, atom_map, "am1bcc", partial_charges)
-    assert np.allclose(openff_mol.partial_charges.magnitude, partial_charges)
+    assert_allclose(openff_mol.partial_charges.magnitude, partial_charges)
```

Comment:
Wondering what is the difference here? (thought the original syntax is more consistent with the `assert condition` syntax)

From the [assert_allclose documentation](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html), these two seems equivalent:
>The test is equivalent to allclose(actual, desired, rtol, atol) (note that allclose has different default values). It compares the difference between actual and desired to atol + rtol * abs(desired).

Reply:
compare the error messages when they each fail. `assert_allclose` gives a much more informative message

Reply:
Thanks for the input! Appreciate that.